assets/js/hooks/events/handler.ts

```diff
@@ -2,39 +2,40 @@ import { NodeData, UserAction } from "../types";
 import {
   getNodeById,
   moveNode,
-  setValue,
+  setData,
+  setContent,
+  focusNode,
   addEditingUserLabel,
   removeEditingUserLabel,
 } from "../node";
 
 export function handleBlur({ uuid, user_name }: UserAction) {
-  const node = getNodeById(uuid)!;
-  removeEditingUserLabel(node, user_name);
+  // const node = getNodeById(uuid)!;
```

Comment:
why not delete?

Reply:
in the line below (line 14) we call: `removeEditingUserLabel(user_name);`
this function does remove the user-label (if exists):
```
removeEditingUserLabel(user_name: string) {
  const span = document.getElementById(user_name);
  span && span.remove();
}
```